README.md

```diff
@@ -73,7 +73,7 @@ It integrates the following projects:
 
 ## Credits
 
-I’m not working alone on this project. First, many people create PR for this projects. Then, there is the whole #pinetime community : a lot of people all around the world who are hacking, searching, experimenting and programming the Pinetime. We exchange our ideas, experiments and code in the chat rooms and forums.
+I’m not working alone on this project. First, many people create PR for this project. Then, there is the whole #pinetime community : a lot of people all around the world who are hacking, searching, experimenting and programming the Pinetime. We exchange our ideas, experiments and code in the chat rooms and forums.
```

Comment:
```suggestion
I’m not working alone on this project. First, many people create PR for this project. Then, there is the whole #pinetime community : a lot of people all around the world who are hacking, searching, experimenting, and programming the Pinetime. We exchange our ideas, experiments, and code in the chat rooms and forums.
```
add commas in lists

Reply:
I was taught not to add commas before and, also this is not related to the changes in the PR.

Reply:
The Oxford Comma, one of the most contentious issues in the English language. I think we should only add a comma if it is necessary to resolve ambiguity. Since this isn't ambiguous, I think it's fine how it is.

Reply:
I apologize, I was always taught that the comma before an 'and' was always necessary. I did not know both were acceptable until now.